.github/workflows/extract-locales.yml

```diff
@@ -14,6 +14,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Should we have that in all our workflows btw ?

Reply:
I don't think so. It's only in this one so we can commit on the branch (even dry mode requires a writeable branch) but PR checkout by default in a headless branch.

In other workflows we don't need to commit so it's fine to checkout the PR. Or is there some other reason we'd want the branch to be not headless?

Reply:
Sorry I was thinking bout specifying the depth, not setting it to 0 for the other workflows - but looking at the docs, the default is 1, so we don't need to do anything elsewhere, ignore me